modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-commit.adoc

```diff
@@ -0,0 +1,43 @@
+= Use with GCP Commits
+:description: Subscribe to Redpanda Cloud in Google Cloud Marketplace with discounted commitment billing.
+
+With a usage-based billing commitment, you sign up for a minimum spend amount and get a discount on your committed usage. This allows you to subscribe to Redpanda Cloud through Google Cloud Marketplace to quickly provision Redpanda Dedicated Cloud clusters. 
+
+Commitments are minimums. If you use more than your committed amount, you can continue using Redpanda Cloud without interruption. You will be charged at your discounted rate for usage beyond the committed amount until the end of your commitment term. 
+
+[IMPORTANT]
+====
+When you sign up for Redpanda Cloud through Google Cloud Marketplace, you can only create clusters on GCP. 
+====
+
+== Sign up in Google Cloud Marketplace
+
+. Contact https://redpanda.com/contact[Redpanda sales^] to request a quote for a GCP Redpanda Cloud commitment. You will receive a private quote.
+
+. Click the private quote link to view the quote on the GCP Marketplace. Review the policy and required terms, and click *Accept*.
++
+[NOTE]
+====
+If you don't have a billing account associated with your project, you're prompted to enable billing to link the subscription with a billing account.
+====
++
+You are taken to the Redpanda sign-up page.
+
+. On the Redpanda sign-up page: 
+* For **Email**, enter your email address to register with Redpanda.
+* For **Organization name**, enter a name for your new organization connected through Google Cloud Marketplace. Redpanda organizations contain all resources, including clusters and networks. You can rename the organization name later in Redpanda Console. 
```

Comment:
is it really possible to rename org? And through console? I thought console does not really interact with organizations

Reply:
Removed "You can rename the organization name later in Redpanda Console."